src/components/ShareStructure/ShareStructure.vue

```diff
@@ -719,6 +752,35 @@ export default class ShareStructure extends Vue {
     this.showSeriesEditForm = [false]
     this.parentIndex = -1
     this.nextId = -1
+    this.scrollToTop(this.$el)
+  }
+
+  private confirmShareRemoval (index: number): void {
+    const shareClass = { ...this.getShareClasses[index] }
+
+    if (shareClass.series.length > 0) {
+      // open confirmation dialog and wait for response
+      this.$refs.confirm.open(
+        'Remove Share Series with Class',
+        'A share series exists for this class. Removing the share class will remove all associated share ' +
```

Comment:
Ticket 5305 mentions a missing dialog when turning off Special Rights & Restrictions. Where is this new dialog?

Reply:
Good question, the ticket was a bit misleading. 
There was no 'new' dialog per se, they just modified the text they wanted to see in the different scenarios in different places.

Reply:
Ah.
Could you please update the ticket then? Use markdown ~strikeout~ style as necessary.
Also I'm sure Riyaz would love a "Test Notes" section/comment in your ticket of what to verify.

Reply:
For sure! 
The design ticket attached is also very descriptive on exactly what they wanted, by which he will have something very concrete to reference.

Reply:
True. May not be needed then.